scripts/validate-instance-configuration.tsx

```diff
@@ -1,60 +1,117 @@
 import React from 'react';
 import {Modal} from 'superdesk-ui-framework/react';
 import ng from 'core/services/ng';
-import {gettext} from 'core/utils';
+import {gettext, gettextPlural} from 'core/utils';
 import {showModal} from '@superdesk/common';
-import {authoringReactEnabledUserSelection, extensions} from 'appConfig';
+import {appConfig, authoringReactEnabledUserSelection, extensions} from 'appConfig';
 import {flatMap} from 'lodash';
 
 interface IError {
     title: string;
     content: React.ReactNode;
 }
 
-function getError(): Promise<IError | null> {
+const Heading = (props: {text: string}) => (
+    <h4 style={{paddingBlockEnd: 'var(--gap-1)'}}>
+        {props.text}
+    </h4>
+);
+
+async function getError(): Promise<IError | null> {
+    const issues: Array<React.ReactNode> = [];
+
     const authoringHeaderComponentsFromExtensions = flatMap(
         Object.values(extensions),
         (extension) => extension.activationResult?.contributions?.authoringHeaderComponents ?? [],
     );
 
     if (authoringHeaderComponentsFromExtensions.length > 0 && authoringReactEnabledUserSelection === true) {
-        return Promise.resolve({
-            title: 'Incompatible extension detected',
-            content: (
-                <>
-                    <p>
-                        Unsupported extension point is being used: `contributions.authoringHeaderComponents`
-                    </p>
+        issues.push((
+            <>
+                <Heading
+                    text={'Unsupported extension point is being used: `contributions.authoringHeaderComponents`'}
+                />
+
+                <p>
+                    You are likely running an outdated version of auto tagging extension.
```

Comment:
seems like it's mixing translated strings with english ones, I wouldn't mind english only here, or having all translated, but wouldn't mix it